lib/uplink/bucket_attrs_test.go

```diff
@@ -53,6 +54,86 @@ func testPlanetWithLibUplink(t *testing.T, cfg testConfig,
 	})
 }
 
+// check that partner bucket attributes are stored and retrieved correctly.
```

Comment:
Rational for code Changes: This test case was dropped during the merge. I added back the change and also improved the test case by making meaningful comparision between expected and actual.